spring-cloud-alibaba-dependencies/pom.xml

```diff
@@ -328,4 +328,37 @@
         </profile>
 
     </profiles>
+
+    <repositories>
```

Comment:
Does this need to be added? We generally define it in the settings.xml file of maven.

Reply:
If definition in local settings.xml, every developer needs to know which repository need to be added .
I think it's necessary before Spring Cloud push the release version , and after Spring Cloud releases the `release` version, i will remove it .